src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.tsx

```diff
@@ -57,7 +57,7 @@ export default function ArrivalByTimeChart({
                   bottom: 5,
                 }}>
                 <CartesianGrid strokeDasharray="3 3" />
-                <XAxis dataKey="gtfs_route_date" />
+                <XAxis dataKey="gtfs_route_date" reversed={true}/>
```

Comment:
according to prettier - there is missing space before closing space
![image](https://github.com/hasadna/open-bus-map-search/assets/1336862/3aed0dee-9cc2-4a48-84ec-b203d75f576c)

Reply:
Oh. Fixing ASAP!

Reply:
You can run 'yarn lint' to see the linter outputs :)